app/controllers/users.js

```diff
@@ -1,9 +1,30 @@
+const jwt = require('jsonwebtoken');
+
+const config = require('../../config');
+const { expiresIn } = require('../constants');
+
 const { signUp } = require('../mappers/users');
-const { saveUser } = require('../services/users');
+const { saveUser, verifyCredentials } = require('../services/users');
 
 exports.createUser = (req, res) => {
   const dataUser = signUp(req.body);
   saveUser(dataUser)
     .then(response => res.status(200).send(response))
     .catch(err => res.status(400).send(err));
 };
+
+exports.signInUser = (req, res) => {
+  verifyCredentials(req.body.email, req.body.password)
+    .then(response => {
```

Comment:
I think using .then is fine but it tends to generate a lot of extra code. Have you considered using async and await more often over .then? I think using await on top of .then makes the code look a lot better